templates/contract-deploy/run-contract-setup.sh

```diff
@@ -83,10 +83,14 @@ fi
 
 # If we had a genesis, and combined that were created outside of
 # kurtosis entirely, we'll use those and exit. This is like a
-# permissionless use case
+# permissionless use case or a use case where we're starting from
+# recovered network
 if [[ -e "/opt/contract-deploy/genesis.json" && -e "/opt/contract-deploy/combined.json" ]]; then
     echo_ts "We have a genesis and combined output file from a previous deployment"
     cp /opt/contract-deploy/* /opt/zkevm/
+    pushd /opt/zkevm || exit 1
+    jq '.firstBatchData' combined.json > first-batch-config.json
```

Comment:
We should also generate the other cdk-erigon configs here no? That's something I've broken into two scripts in my PR (#361): 1) deploy zkevm contracts, 2) generate cdk-erigon allocs, config and first batch config.

Reply:
That's an interesting point. It's worth skimming over the document to see exactly what I was doing. I'm 100% sure there are better ways. In my case, I used the `genesis.json` and `combined.json` files to create the erigon files. They're already going to be in the `/opt/contract-deploy` directory. 

Perhaps at a minimum, I sure ensure those files are in the directory before preceding?

Reply:
Now that I think about it, we proably don't need to add explicit check her because if we're using erigon, the templating starlark code would fail if something is wrong and if we're not using erigon, there's no point checking.